server/api/users/users.policy.js

```diff
@@ -36,19 +70,39 @@ exports.serialize = function(req, user) {
     email: user.get('email')
   };
 
-  const admin = req.currentUser && req.currentUser.hasRole('admin');
-  const sameUser = req.currentUser && req.currentUser.get('api_id') === user.get('api_id');
+  const isAdmin = req.currentUser && req.currentUser.hasRole('admin');
+  const isSameUser = req.currentUser && req.currentUser.get('api_id') === user.get('api_id');
 
-  if (admin || sameUser) {
+  // User properties visible to administrators and to the user itself.
+  if (isAdmin || isSameUser) {
     _.extend(serialized, {
       id: user.get('api_id'),
       href: user.get('href'),
+      firstName: user.get('first_name'),
+      lastName: user.get('last_name'),
+      emailVerified: user.get('email_verified'),
+      emailVerifiedAt: user.get('email_verified_at'),
       active: user.get('active'),
+      provider: user.get('provider'),
+      providerId: user.get('provider_id'),
       roles: user.get('roles'),
       createdAt: user.get('created_at'),
       updatedAt: user.get('updated_at')
     });
   }
 
-  return serialized;
+  // Properties visible only to administrators.
+  if (isAdmin) {
+    _.extend(serialized, {
+      providerData: user.get('provider_data') || {},
+      registrationOtp: user.get('registration_otp'),
```

Comment:
Administrators should be able to see everything for troubleshooting. It can also be useful if we have temporary email sending problems on the server: the administrator can register a user and send the email manually.